packages/-ember-data/tests/integration/adapter/record-persistence-test.js

```diff
@@ -28,7 +28,7 @@ module('integration/adapter/record_persistence - Persisting Records', function(h
     this.owner.register('serializer:application', JSONAPISerializer.extend());
   });
 
-  test("When a store is committed, the adapter's `commit` method should be called with records that have been changed.", function(assert) {
+  test("When a store is committed, the adapter's `updateRecord` method should be called with records that have been changed.", async function(assert) {
```

Comment:
the small improvement to this test file is not related to the rest of the PR, just slipped in as a teaching example while pairing on the wider PR. happy to separate out if desired but seemed small and a happy cleanup.

Reply:
❤️ for the note